### PR TITLE
[MIG] [16.0] maintenance_plan_activity: migration to 16.0

### PR DESCRIPTION
**Pull request by luisDIXMIT**
Standard migration

@etobella

### CODE REVIEW

File: maintenance_plan_activity/__manifest__.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
         This module allows defining in the maintenance plan activities that
         will be created once the maintenance requests are created as a
         consequence of the plan itself.""",
-    "version": "15.0.1.0.2",
+    "version": "16.0.1.0.2",
```
Comment (etobella) [inline maintenance_plan_activity/__manifest__.py]: Keep it 16.0.1.0.0

On new odoo versions, we start from 1.0.0 again

**Review by etobella — APPROVED** [review]
Tested on runboat

No major changes

/ocabot migration maintenance_plan_activity

**Review by mamcode — APPROVED** [review]
LGTM 👍🏼

**Pull request merged**